lms/static/js/views/fields.js

```diff
@@ -116,8 +141,8 @@
             },
 
             showSuccessMessage: function () {
-                var successMessage = this.getMessage('success');
-                this.showNotificationMessage(successMessage);
+                var successMessage = this.getMessage('success').toString();
```

Comment:
Currently, `this.getMessage()` returns a HtmlSnippet while `view.getNotificationMessage()` returns a string. Additionally, `showNotificationMessage()` takes a HtmlSnippet  which is why line 145 has the repeated `getMessage()` call since line 144 has the `toString()` call. 

1. I could have both `getMessage()` and `getNotificationMessage()` return a **string**, but then `showNotificationMessage()` would have to change all strings to HtmlSnippets.

2. I could have both `getMessage()` and `getNotificationMessage()` return a **HtmlSnippet**, but the functions sound like they should return strings instead of HTML code.

Which would you guys prefer?